src/api/app/controllers/trigger/errors.rb

```diff
@@ -1,4 +1,8 @@
 module Trigger::Errors
+  class NotEnabledToken < APIError
```

Comment:
What about `DisabledToken`?

Reply:
> What about `DisabledToken`?

No strong opinion. Any particular reason you think it should be changed...?

Reply:
`NotEnabled` is negative logic, is like using `if !missing?` instead of `if found?`.

Reply:
Sure. `NotEnabled` is negative logic. We have no preference in using negative logic or positive logic, I guess.

Also, `NotEnabled` helps to understand better the code internally. It makes more clear a reference to an internal `enabled` field.

Reply:
> Sure. `NotEnabled` is negative logic. We have no preference in using negative logic or positive logic, I guess.

Well, yes we do, we do not use `if !false` or anything like this. We go for the simple things, and coding in positive logic is the simple way.

Reply:
> Well, yes we do, we do not use `if !false` or anything like this. We go for the simple things, and coding in positive logic is the simple way.

I totally agree with coding. However, variable/exception naming is the subject here...

Reply:
Naming variables [is coding](https://martinfowler.com/bliki/TwoHardThings.html)! Anyway, I don't have any more arguments in favor of using `DisabledToken` :shrug:

Reply:
> Naming variables [is coding](https://martinfowler.com/bliki/TwoHardThings.html)! Anyway, I don't have any more arguments in favor of using `DisabledToken` 🤷

Let's keep `NotEnabledToken` then, only for this time. :smile: